### DISCUSSION TITLE
Hide Achievements

### DISCUSSION THREAD

**Discussion by Committing** [community/community#18171] [Other Feature Feedback, Questions, & Ideas] [answered]
Option to toggle on/off profile achievement visibility:

Badges take away professionalism.

Badges look weak.

Badges point out uninteresting logic.

Badges take away from desired points of view of a person's profile.

Badges stand out too much.

Badges are unnecessary.

Badges make irrelevant points.

Badges look childish.

Badges are cringy.

**Comment by Committing** [comment, marked as answer]
OK:

Settings / Public Profile / Show Achievements on my profile, uncheck it.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
